server/services/allocationsService.ts

```diff
@@ -31,4 +31,12 @@ export default class AllocationsService {
       headers: { Accept: 'application/json' },
     })) as Allocation[]
   }
+
+  async getUnallocatedCase(token: string, crn): Promise<Allocation> {
+    logger.info(`Getting unallocated case for crn ${crn}`)
+    return (await this.restClient(token).get({
```

Comment:
Good work on doing this, could the story be demoed without calling this endpoint? maybe always return a dummy instance of Allocation with the same name? Then when we loop round to integrating with this endpoint we know we can return the Allocation payload and there is one less factor in the integration